src/main/java/com/kk/strong/controller/UserController.java

```diff
@@ -0,0 +1,33 @@
+package com.kk.strong.controller;
+
+import com.kk.strong.model.User;
+import com.kk.strong.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class UserController {
```

Comment:
I saw that you use the approach of securing endpoints in `SecurityConfig`, there is also approach of roles - something like this `@PreAuthorize("hasRole('ROLE_ADMIN')")` please check this as the more roles you have in the application, the more complex security config is and with this annotation you can see in controller who has access and who don't